src/anm/animation/element.js

```diff
@@ -805,14 +813,13 @@ Element.prototype.render = function(ctx) {
             this.transform(bctx);
             this.painters(bctx);
             this.each(function(child) {
-                child.tick(dt);
                 child.render(bctx, dt);
             });
 
+            // FIXME: this should be performed one time for all masked elements!
             mask.transform(mctx);
```

Comment:
mask canvas size does not change during one rendering cycle. the masked elements and the mask itself are rendered into canvases the same size.

you need to draw all masked elements into a canvas first, then (after all masked elements are rendered) you need to render mask itself